### DISCUSSION TITLE
How do I test the vid4 dataset using basicvsr++

### DISCUSSION THREAD

**Discussion by 11923303233** [open-mmlab/mmediting#1283] [How-to]
Should I use SRVidDataset or SRFolderMultiGTDataset

**Comment by 11923303233** [comment]
When I use the pretrained weights of basicvsr++_c64n7_8x1, use SRFolderMultipleGTDataset, num_input_frames = None, the PSNR obtained is inconsistent with the official one

**Comment by 11923303233** [comment]
> Should I use SRVidDataset or SRFolderMultiGTDataset

or SRtestMultipleDataset

**Comment by zengyh1900** [comment]
@Z-Fran 
Please check this issue.
